src/main.py

```diff
@@ -38,20 +39,19 @@ def print_exception(sanitize: bool) -> None:
 
 
 def run(options: Options) -> int:
-    mips_file: MIPSFile
```

Comment:
while Python 3.6 doesn't have ordered dicts in its formal spec, all implementations of it do. I think we rely on that in other places already